src/App.js

```diff
@@ -53,7 +56,7 @@ function App() {
             <Route exact path="/">
               <Input input={input} inputHandler={inputHandler} />
```

Comment:
- [ ] I cannot find the `input.js` and `inputButton.js` files in the changed files. So allow me to make the comments here. The Input and InputButton components are better if wrapped inside a form component. In addition, you need at least two input elements for the book title and book author as the book object has a title and author props. :pray:  Kindly read the [project's requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_create_and_remove_book.md) carefully.

Reply:
- [ ] As mentioned by the previous reviewer in addition to the book title you should also have book author because it is a requirement